Documentation/local-kpm-registry.md

```diff
@@ -0,0 +1,68 @@
+## Running a local registry
+
+You may want to run a kpm registry locally, and then deploy your kpm registry to k8s from this "bootstrapping registry" if-you-will.
+
+It requires etcd to be present, so get up etcd first.
+
+```
+$ docker run --name tempetcd -dt -p 2379:2379 -p 2380:2380 quay.io/coreos/etcd:v3.0.6 /usr/local/bin/etcd -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -advertise-client-urls http://$127.0.0.1:2379,http://127.0.0.1:4001
```

Comment:
There is two new backend, easier to setup than etcd: filesystem and redis
 With filesystem it creates an index and store blob on disk, so no extra service required. 
I'm using it for dev. 

DATABASE_URL=path-on-disk STORAGE=filesystem 
```
$ PORT=5000 DATABASE_URL="$HOME/.kpm/packages"  STORAGE=filesystem ./run-server.sh
[server-logs]
$ cd deploy/kpm-registry/
$ kpm push localhost:5000
package: coreos/kpm-registry (0.24.1-1 | kpm) pushed

$ tree ~/.kpm/packages/cnr/packages                                                                                                             
/home/ant31/.kpm/packages/cnr/packages
├── coreos
│   └── kpm-registry
│       ├── digests
│       │   └── sha256
│       │       └── b4eb1e5fbb1f251a97b7653f866cb482718ac96dd6d30b35f4522b705dd6f76e
│       └── releases.json
└── packages.json
$ kpm list localhost:5000
app                  release    downloads    manifests
-------------------  ---------  -----------  -----------
coreos/kpm-registry  0.24.1-1   -            kpm
```